python/lsst/sconsUtils/tests.py

```diff
@@ -297,6 +313,81 @@ def run(self, fileGlob):
 
         return targets
 
+    def runPythonLinter(self):
+        """Add a target for running the python linter.
+
+        Returns
+        -------
+        target : `list`
+            Returns a list containing a single target. Can be empty
+            if no suitable linter configuration was found.
+        """
+        linter: str | None = None
+        root = SCons.Script.Dir("#").abspath
+
+        pyproject = os.path.join(root, "pyproject.toml")
+        if os.path.exists(pyproject):
+            import tomli
```

Comment:
So we're going to use `tomli` for backward compat but maybe move to `tomllib` when we have a 3.11 minimum?

Reply:
No. I hadn't realized it was renamed to `tomllib` and since this can only merge after we go to rubin-env 7 there is no reason to try to continue to use `tomli`.